tests/components/prometheus/test_init.py

```diff
@@ -404,29 +446,30 @@ async def test_humidifier(
     """Test prometheus metrics for humidifier entities."""
     body = await generate_latest_metrics(client)
 
-    assert (
-        'humidifier_target_humidity_percent{domain="humidifier",'
-        'entity="humidifier.humidifier",'
-        'friendly_name="Humidifier"} 68.0' in body
+    MetricsTestHelper._perform_humidifier_metric_assert(
```

Comment:
These device_class specific assertion methods seem like an unnecessary layer of abstraction. The `_perform_humidifier_metric_assert` is only used a handful of times here - it might be simpler to use a generic function, or have a helper function that's defined closer to where it's used.